clean.cmd

```diff
@@ -61,9 +61,9 @@ shift /1
 goto Loop
 
 :Begin
-
-echo Running init-tools.cmd
-call %~dp0init-tools.cmd
+REM If VBCSCompiler.exe is running we need to kill it
+tasklist /fi "imagename eq VBCSCompiler.exe" |find ":" > nul
+if errorlevel 1 taskkill /f /im "VBCSCompiler.exe"
```

Comment:
Rather than kill it, have you tried passing -shutdown to gracefully stop the process?